MAVProxy/mavproxy.py

```diff
@@ -697,7 +738,7 @@ def main_loop():
         for m in mpstate.mav_outputs:
             rin.append(m.fd)
         if rin == []:
-            time.sleep(0.0001)
+            time.sleep(0.1) # We have no ports, check back again in 100ms
```

Comment:
is there a way to identify joystick so we can add a conditional or any other type of check?